### ISSUE TITLE
Lane should be identified using NonNegativeInteger (Bugzilla Bug 275)

### ISSUE THREAD

**Issue by datexii** [DATEX-II-EU/DatexII#275]
This issue was created automatically with bugzilla2github.py
# Bugzilla Bug 275
Date: 2019-09-19T10:22:11+02:00
From: Ian Snell &lt;<ian.snell@siemens.com>&gt;
To:   Jean-Philippe Mechine &lt;<Jean-Philippe.Mechin@cerema.fr>&gt;

Last updated: 2019-09-19T10:22:11+02:00

**Comment by LBlaive** [comment]
During the meeting of the 26th September 2024 it was agreed that:!
- The data type assigned to "laneNumber" is set at "NonNegativeInteger";
- The current definition is completed to cover the value 0
- New text: "The number of the lane, where 1 i**dentifies the nearest lane of the** hard shoulder/verge and the numbers increase towards the central reservation/road axis. **The value "0" may be allocated to the outer hard shoulder when it is wide enough to be used as emergency lane.**"
- Addition to the "hardShoulder" literal (in "LaneEnum"): a comment will be added to explain than in Europe hard shoulder is generally used to name the outer hard shoulder.
-

**Comment by iancornwell1** [comment]
I agree except that inside the new text "the nearest lane of the hard shoulder/verge" is not natural English. The smallest change to make it ok is changing "of" to "to", but it would be better still to change the word order from "nearest lane to" to "lane nearest to", and having done that the "to" becomes totally optional in English so the original v3.X wording "the lane nearest the hard shoulder/verge" is also fine.

**Comment by LBlaive** [comment]
Being a non-native speaker I totally agree with you.
